docker/docker.go

```diff
@@ -515,6 +516,8 @@ func IsContainerUpgradable(ctx context.Context, dClient *client.Client, imageCfg
 }
 
 func sliceEqualsIgnoreOrder(left, right []string) bool {
+	logrus.Debugf("Comparing, left: %s", left)
```

Comment:
This will generate tons of debug messages and will not really be useful to see which lists are we comparing, can we omit this

Reply:
It would make sense to log in debug when the result of comparison is false, so it's easier to figure out the discrepancy.

Reply:
This was scoped to checking if it changed, if so, it will log the diff not both the values.